Use new-style backquotes, to make it compile with Emacs 27.

--- gnuserv-3.12.8-orig/gnuserv-compat.el
+++ gnuserv-3.12.8/gnuserv-compat.el
@@ -193,9 +193,9 @@
     (defmacro defgroup (&rest args)
       nil)
     (defmacro defcustom (var value doc &rest args)
-      (` (defvar (, var) (, value) (, doc))))
+      `(defvar ,var ,value ,doc))
     (defmacro defface (var value doc &rest args)
-      (` (make-face (, var))))
+      `(make-face ,var))
     (defmacro define-widget (&rest args)
       nil)))
 
